encrypted-media-respec.html

```diff
@@ -1169,7 +1169,7 @@
         <dd>
           <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/85">Issue 85</a> - There are concerns about the architectural implications of <code>"persistent-usage-record"</code> sessions that are pending a TAG discussion. The outcome of this discussion could result in modification (including removal) of the feature.</p>
           <p>
-            A session for which the license and any key(s) it contains SHALL NOT be persisted and for which a <a def-id="record-of-key-usage"></a> SHALL be persisted when
+            A session for which the license and key(s) are not persisted and for which a <a def-id="record-of-key-usage"></a> is persisted when
```

Comment:
Yes, the normative requirement is moved to the _Destroy keys and licenses_ algorithm (Section 6.6.5) and so this text becomes descriptive so as not to have duplicate specification of the same normative requirement.

And yes, it would make sense to move the requirement to store the record of license destruction for `persistent` sessions as well.